### PR TITLE
Dashboard changes and navbar changed

### PR DESCRIPTION
**Pull request by Manikanta662001**

### CODE REVIEW

File: src/pages/dashboard/AdminRequestMessages.tsx [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+interface ChatRequestInterface {
+  date: string;
+  isPending: boolean;
+  opponent: { name: string; id: string };
+  sender: { name: string; id: string };
+  time: string;
+  __v: number;
+  _id: string;
+}
+interface TicketRequestInterface {
```
Comment (ResourceOne2011) [inline src/pages/dashboard/AdminRequestMessages.tsx:14]: move these 2 interfaces to the modals folder and create a messagemodal file

File: src/pages/dashboard/AdminRequestMessages.tsx [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+    TicketRequestInterface[]
+  >([]);
+  useEffect(() => {
+    httpMethods
+      .get<any>("/message/user-chat-request")
+      .then((res: ChatRequestInterface[]) => {
+        setChatRequests(res);
+      })
+      .catch((errr) => {
+        throw new errr();
```
Comment (ResourceOne2011) [inline src/pages/dashboard/AdminRequestMessages.tsx:36]: why u r throwing err , just store it in one state variable and bind somewhere in the component , don't use throw like this

File: src/pages/dashboard/userDashboard/UserDashboard.tsx [hunk 3]
```diff
@@ -173,7 +157,7 @@ const UserDashboard = ({ user }: { user: UserModal }) => {
   return (
     <>
       <div className="userdashboard">
-        <div className="user-nav-header">
+        {/* <div className="user-nav-header">
```
Comment (ResourceOne2011) [inline src/pages/dashboard/userDashboard/UserDashboard.tsx:160]: remove the commented code

File: src/pages/tickets/TicketDescription.tsx [hunk 4]
```diff
@@ -17,25 +19,26 @@ const TicketDescription = () => {
...
+          <b>Description :</b> {selectedTicket.description}
         </p>
         <p>
-          <b>Comments :</b> {state.comments}
+          <b>Comments :</b> {selectedTicket.comments}
         </p>
         <p>
-          <b>ReceivedDate :</b> {state.receivedDate}
+          <b>ReceivedDate :</b>{" "}
+          {selectedTicket.receivedDate as unknown as string}
```
Comment (ResourceOne2011) [inline src/pages/tickets/TicketDescription.tsx:41]: receivedDate type is Date so use new Date(receivedDate) like this

File: src/pages/tickets/TicketDescription.tsx [hunk 5]
```diff
@@ -44,7 +47,7 @@ const TicketDescription = () => {
         )}
 
         <ul className="all-desc">
-          {state.updates?.map((item: any, index: number) => {
+          {selectedTicket.updates?.map((item: any, index: number) => {
```
Comment (ResourceOne2011) [inline src/pages/tickets/TicketDescription.tsx:50]: remov the optional chaining

File: src/utils/Constants.tsx [hunk 6]
```diff
@@ -3,5 +3,5 @@ import { cookieComp } from "./utils";
 const BE_SERVER = "https://task-tracker-server-2njm.onrender.com";
 const BE_LOCAL = "http://192.168.10.30:1234";
 const BE_LOCAL2 = "http://192.168.29.110:1234";
-export const BE_URL = BE_SERVER;
+export const BE_URL = BE_LOCAL;
```
Comment (ResourceOne2011) [inline src/utils/Constants.tsx:6]: change it to server every time u r pushing don't push this file

File: src/pages/tickets/TicketDescription.tsx (hunk 5 above)
Comment (Manikanta662001) [reply src/pages/tickets/TicketDescription.tsx:50]: done

File: src/utils/Constants.tsx (hunk 6 above)
Comment (Manikanta662001) [reply src/utils/Constants.tsx:6]: done

File: src/pages/dashboard/userDashboard/UserDashboard.tsx (hunk 3 above)
Comment (Manikanta662001) [reply src/pages/dashboard/userDashboard/UserDashboard.tsx:160]: done

File: src/pages/dashboard/AdminRequestMessages.tsx (hunk 2 above)
Comment (Manikanta662001) [reply src/pages/dashboard/AdminRequestMessages.tsx:36]: ok

File: src/pages/dashboard/AdminRequestMessages.tsx (hunk 1 above)
Comment (Manikanta662001) [reply src/pages/dashboard/AdminRequestMessages.tsx:14]: done

File: src/pages/dashboard/AdminRequestMessages.tsx [hunk 7]
```diff
@@ -0,0 +1,82 @@
...
+      .catch((errr: string) => {
+        alert(errr);
+      });
+    httpMethods
+      .get<any>("/message/user-ticket-request")
+      .then((res: any) => {
+        setTicketRequests(res);
+      })
+      .catch((errr) => {
+        throw new errr();
```
Comment (ResourceOne2011) [inline src/pages/dashboard/AdminRequestMessages.tsx:30]: remove this error

File: src/pages/dashboard/AdminRequestMessages.tsx [hunk 8]
```diff
@@ -0,0 +1,82 @@
...
+                  )}
+                </p>
+              </div>
+            );
+          })}
+        </div>
+        <div className="request-sub-msg">
+          <h3>Ticket Requests</h3>
+          {ticketRequests?.map((ticket) => {
+            return (
```
Comment (ResourceOne2011) [inline src/pages/dashboard/AdminRequestMessages.tsx:60]: render direct using parenthesis and remove the return method

**Review by ResourceOne2011 — APPROVED** [review]

File: src/pages/dashboard/AdminRequestMessages.tsx (hunk 7 above)
Comment (Manikanta662001) [reply src/pages/dashboard/AdminRequestMessages.tsx:30]: done

**Pull request merged**